src/clarity-angular/utils/conditional/if-open.directive.ts

```diff
@@ -68,7 +70,13 @@ export class IfOpenDirective implements OnInit {
     }
 
     constructor( private ifOpenService: IfOpenService, private template: TemplateRef<any>,
```

Comment:
Is the subscription to services in the constructor the preferred way to listen to observables?
It seems like we are moving the subscriptions here instead of OnInit. (I think I saw similar changes in the Chocolate PR too)
 /cc @youdz